dockerfiles/docker-compose.yml

```diff
@@ -98,7 +101,7 @@ services:
     tty: true
     networks:
       readthedocs:
-    command: ["../../docker/celery.sh"]
+    command: ["../../docker/wait-for-it.sh", "search:9200", "--timeout=360", "--", "../../docker/celery.sh"]
```

Comment:
I think this may be a problem. What happens when we call `inv docker.up --no-search`? I guess it will wait and timeout before running `celery.sh`, right?

The script we used before handles this properly by taking care of `--no-search` checking ` if 'SEARCH' in os.environ` before waiting for search.

Reply:
Ay, thank you for catching that, will fix!

Reply:
This is fixed and tested that `celery` comes up immediately with `--no-search`